src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($day-width * 7) + ($gap * 6) + ($padding * 2);
+  height: 100%;
+  padding: $padding;
+  gap: $gap;
+  justify-content: flex-start;
```

Comment:
Pay attention to your CSS properties order
https://9elements.com/css-rule-order/